docs/public/configuration/backend.md

```diff
@@ -59,31 +60,57 @@ In addition to the configuration file, some settings can configured using
 
 ## RPCAPI section
 
-Available keys: `enable_batch_create`, `enable_user_create`,
-`enable_add_batch_job`, `enable_add_api_user`.
+Available keys: `batch_api_key`, `enable_add_batch_job`, `enable_add_api_user`,
+`enable_batch_create`, `max_batch_size_non_auth`.
+
+### batch_api_key
+
+API key to use in requests to be granted `trusted` privilege level. E.g., to
+allow creation of larger batches using the `batch_create` method.
+
+A string of alphanumerics of at least 10 and at most 64 characters. I.e. a string
+matching `/^[a-zA-Z0-9]{10,64}$/`. It is recommended to use an `batch_api_key` of
+at least 22 characters.
+
+Multiple `batch_api_key` is supported. All configured keys will be equal in
+authorization.
```

Comment:
The syntax for multiple API keys should be specified.  For `LANGUAGE.locale` we're using a space separated list. That should work here too.